src/components/CustomTag/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Typography } from "@material-ui/core";
+import React, { useEffect } from "react";
+import ReactMarkdown from "react-markdown";
+import { BLOG_KEYWORDS } from "../../constants";
+import useStyles from "./styles";
+
+interface blogTitleProps {
+  blogLabel: string;
+}
+
+const CustomTag: React.FC<blogTitleProps> = ({ blogLabel }) => {
+  const classes = useStyles();
+  const handleTags = () => {
+    if (blogLabel === BLOG_KEYWORDS.CHOAS_ENGINEERING) {
```

Comment:
If at all this block is needed, let's use a switch case instead of multiple layers of if .. else if

Reply:
Updated code block using switch